docs/about/release-notes.rst

```diff
@@ -21,6 +21,9 @@ Bugfixes
 ~~~~~~~~
```

Comment:
I guess there are plenty of places in the docs where we can also replace the old syntax with the new, but I assume you were leaving that to a follow-up PR?

Reply:
Yes, follow-up. I think it is not just replacing old uses, but also add some new sections, possibly.